sql/mob_skill_lists.sql

```diff
@@ -3461,3 +3461,24 @@ INSERT INTO `mob_skill_lists` VALUES ('Siren',1010,3512); -- Sonic Buffet
 INSERT INTO `mob_skill_lists` VALUES ('Siren',1010,3513); -- Entice
 INSERT INTO `mob_skill_lists` VALUES ('Siren',1010,3514); -- Hysteric Assault
 INSERT INTO `mob_skill_lists` VALUES ('Siren',1010,3515); -- Clarsach Call
+INSERT INTO `mob_skill_lists` VALUES ('Chaos_Wyvern',1012,813); -- Dispelling Wind
+INSERT INTO `mob_skill_lists` VALUES ('Chaos_Wyvern',1012,814); -- Deadly Drive 
+INSERT INTO `mob_skill_lists` VALUES ('Chaos_Wyvern',1012,815); -- Wind Wall
+INSERT INTO `mob_skill_lists` VALUES ('Chaos_Wyvern',1012,816); -- Fang Rush
+INSERT INTO `mob_skill_lists` VALUES ('Chaos_Wyvern',1012,817); -- Dread Shriek
+INSERT INTO `mob_skill_lists` VALUES ('Chaos_Wyvern',1012,818); -- Tail Crush
+INSERT INTO `mob_skill_lists` VALUES ('Chaos_Wyvern',1012,822); -- Chaos Breath
+INSERT INTO `mob_skill_lists` VALUES ('Thunder_Wyvern',1011,813); -- Dispelling Wind
+INSERT INTO `mob_skill_lists` VALUES ('Thunder_Wyvern',1011,814); -- Deadly Drive
+INSERT INTO `mob_skill_lists` VALUES ('Thunder_Wyvern',1011,815); -- Wind Wall
+INSERT INTO `mob_skill_lists` VALUES ('Thunder_Wyvern',1011,816); -- Fang RUsh
+INSERT INTO `mob_skill_lists` VALUES ('Thunder_Wyvern',1011,817); -- Dread Shriek
+INSERT INTO `mob_skill_lists` VALUES ('Thunder_Wyvern',1011,818); -- Tail Crush
+INSERT INTO `mob_skill_lists` VALUES ('Thunder_Wyvern',1011,820); -- Thunder Breath
```

Comment:
Make sure the name matches the mob name (`Lightning_Wyvern`), and sort the skill list ID (1011) to be above the Chaos Wyvern (1012) and I'll hit the button.